json-api.md

```diff
@@ -39,56 +39,76 @@ Response Code: `200`
 ]
 ```
 
-## Get Configurations
+## Configure SSID Priority
 
-Get the current known configurations.
+This endpoint takes a list of `SSID`s in the order
+of the priority that they should be tried. The
+first element has the highest priority
```

Comment:
"This endpoint takes a list of `SSID`s. Each `SSID` is tried in order until a successful connection is made. It is not required to list all configured `SSID`s."